packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.base.tsx

```diff
@@ -36,15 +53,14 @@ export class ColorPickerBase extends BaseComponent<IColorPickerProps, IColorPick
   private _textChangeHandlers: {
     [K in keyof IRGBHex]: (event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>, newValue?: string) => void
   };
-  private _textBlurHandlers: {
-    [K in keyof IRGBHex]: (event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>, newValue?: string) => void
-  };
   private _textLabels: { [K in keyof IRGBHex]?: string };
 
   constructor(props: IColorPickerProps) {
     super(props);
 
-    this._warnDeprecations({
+    initializeComponentRef(this);
```

Comment:
This line is only needed if you're also getting rid of the BaseComponent inheritance (you could add that part back if you want since it's a non-breaking change, or just remove this line)

Reply:
Removed BaseComponent